### PR TITLE
Minor UI fixes: leave circle error notification, button colors

### PR DESCRIPTION
**Pull request by wingfeatherwave**
Using our below-field errors instead of the native html5 error popover
Screen Shot 2022-10-25 at 11 14 22 AM

### CODE REVIEW

**Comment by levity** [comment]
Maybe while you're in this code you can fix the space after the circle name, "Team" instead of "Circle", and the back tick instead of quote after Team? Thx 🙏🏼

**Comment by wingfeatherwave** [comment]
> Maybe while you're in this code you can fix the space after the circle name, "Team" instead of "Circle", and the back tick instead of quote after Team? Thx 🙏🏼

TY.  Updated more copy with @crabsinger 

Screen Shot 2022-10-25 at 4 23 45 PM

File: src/pages/MembersPage/LeaveCircleModal.tsx [hunk 1]
```diff
@@ -110,7 +110,7 @@ export const LeaveCircleModal = ({
               color="destructive"
               size="large"
               css={{ width: '50%' }}
-              disabled={!isEmpty(errors) || !isDirty}
+              disabled={!isValid}
```
Comment (crabsinger) [inline src/pages/MembersPage/LeaveCircleModal.tsx:113]: @sshmm FYI look at the changes/improvements @wingfeatherwave and I made here. For example, !isValid is a simpler way of doing what you did before.

**Review by crabsinger — APPROVED** [review]

File: src/pages/MembersPage/LeaveCircleModal.tsx (hunk 1 above)
Comment (sshmm) [reply src/pages/MembersPage/LeaveCircleModal.tsx:113]: Got it thanks

**Pull request merged**